README.md

```diff
@@ -46,55 +46,44 @@ A modern, enterprise-ready business intelligence web application.
+- A **lightweight semantic layer** for quickly defining custom dimensions and metrics
+- Out of the box support for **nearly any SQL** database or data engine
+- A wide array of **beautiful visualizations** to showcase your data
+metadata browser
+- Lightweight, configurable **caching layer** to help ease database load
+- Highly extensible **security roles and authentication** options
+- An **API** for programmatic customization
+- A **cloud-native architecture** designed from the ground up for scale
+
 ## Screenshots & Gifs
 
-**Gallery**
+**Large Gallery of Visualizations**
 
 <kbd><a href="https://superset.apache.org/gallery"><img title="Gallery" src="superset-frontend/src/assets/images/screenshots/gallery.jpg"/></a></kbd><br/>
 
-**View Dashboards**
+**Craft Beautiful, Dynamic Dashboards**
 
 <kbd><img title="View Dashboards" src="superset-frontend/src/assets/images/screenshots/slack_dash.jpg"/></kbd><br/>
 
-**Slice & dice your data**
+**No Code Chart Builder**
 
 <kbd><img title="Slice & dice your data" src="superset-frontend/src/assets/images/screenshots/explore.jpg"/></kbd><br/>
 
-**Query and visualize your data with SQL Lab**
+**Powerful SQL Editor**
 
 <kbd><img title="SQL Lab" src="superset-frontend/src/assets/images/screenshots/sql_lab.jpg"/></kbd><br/>
 
-**Visualize geospatial data with deck.gl**
-
-<kbd><img title="Geospatial" src="superset-frontend/src/assets/images/screenshots/geospatial_dash.jpg"/></kbd><br/>
-
-**Choose from a wide array of visualizations**
-
-<kbd><img title="Visualizations" src="superset-frontend/src/assets/images/screenshots/explore_visualizations.jpg"/></kbd><br/>
-
-## Why Superset?
-
-Superset provides:
-
-- An intuitive interface for visualizing datasets and
-  crafting interactive dashboards
-- A wide array of beautiful visualizations to showcase your data
-- Code-free visualization builder to extract and present datasets
-- A world-class SQL IDE for preparing data for visualization, including a rich metadata browser
-- A lightweight semantic layer which empowers data analysts to quickly define custom dimensions and metrics
-- Out-of-the-box support for most SQL-speaking databases
-- Seamless, in-memory asynchronous caching and queries
-- An extensible security model that allows configuration of very intricate rules
-  on who can access which product features and datasets.
-- Integration with major
-  authentication backends (database, OpenID, LDAP, OAuth, REMOTE_USER, etc)
-- The ability to add custom visualization plugins
-- An API for programmatic customization
-- A cloud-native architecture designed from the ground up for scale
-
 ## Supported Databases
 
-Superset can query data from any SQL-speaking datastore or data engine (e.g. Presto or Athena) that has a Python DB-API driver and a SQLAlchemy dialect.
+Superset can query data from any SQL-speaking datastore or data engine (e.g. Presto, Trino, or Athena) that has a Python DB-API driver and a SQLAlchemy dialect.
```

Comment:
Should we just make a bullet list of off-the-shelf connectors, for SEO purposes ad/or general completionisim?

Reply:
We have a list here (https://superset.apache.org/docs/databases/installing-database-drivers/). My general feeling is I want to actually reduce the amount of content in the readme and move more of it to the docs / apache site. This seems consistent anyway with most other open source projects have done (e.g. kafka, spark). As a counter point tho, Airflow still has a bunch of content on their readme.

Maybe for now we just do logos, in my next pass I can enumerate the full list?

Reply:
```suggestion
Superset can query data from any SQL-speaking datastore or data engine (Presto, Trino, Athena, [and more](https://superset.apache.org/docs/databases/installing-database-drivers/)) that has a Python DB-API driver and a SQLAlchemy dialect.
```
Let's use the link, then!